src/FSharpVSPowerTools.Core/OpenDeclarationsGetter.fs

```diff
@@ -460,7 +457,6 @@ module OpenDeclarationGetter =
                     |> Option.bind (function
                          | [||] -> None
                          | prefix -> Some (sUseRange, prefix)))) 
-            |> Array.concat
 
```

Comment:
I was thinking this could be changed to something like

        let private getFullPrefix (longIdents: IDictionary<_,_>) fullName (endPos: Range.pos): Idents option =
            match longIdents.TryGetValue endPos with
            | true, longIdent ->                   
                let rec loop matchFound (longIdents:_[]) (symbolIdents:_[]) (lcnt:int) (scnt:int) =
                    match lcnt, scnt with
                    | lcnt, _    when lcnt = longIdents.Length    -> symbolIdents.[scnt..]
                    | _   , scnt when scnt = symbolIdents.Length  -> [||]
                    | lcnt, scnt ->
                        if longIdents.[lcnt] <> symbolIdents.[scnt] then
                            if matchFound 
                            then symbolIdents.[scnt..] 
                            else loop matchFound longIdents symbolIdents (lcnt+1) (scnt+1)
                        else loop true longIdents symbolIdents (lcnt+1) (scnt+1)
                let prefix = 
                    loop false (longIdent |> Array.rev) (fullName |> Array.rev) 0 0
                    |> Array.rev

but it's probably a bit excessive to get 
```
(longIdent |> Array.rev |> List.ofArray) (fullName |> Array.rev |> List.ofArray)
|> List.rev
|> List.toArray
``` 
to become 
```
loop false (longIdent |> Array.rev) (fullName |> Array.rev) 0 0
|> Array.rev
```